georaster-layer-for-leaflet.js

```diff
@@ -9,6 +9,7 @@ const PROJ4_SUPPORTED_PROJECTIONS = new Set([3857, 4269]);
 const MAX_NORTHING = 1000;
 const MAX_EASTING = 1000;
 const ORIGIN = [0, 0];
+const IS_PRODUCTION = process.env.NODE_ENV === "production";
```

Comment:
In order to play it safe, could we change this to the following?
```js
const IS_PRODUCTION = typeof process === 'object' && typeof process.env === 'object' && process.env.NODE_ENV === "production";
```

I'm wondering if anyone might be trying to directly load `georaster-layer-for-leaflet.js` in the browser where process won't be defined.

Reply:
Ah yes good point, I hadn't considered that. Is there a specific reason you are checking the `typeof` at each step or could we just to: 

```javascript
 const IS_PRODUCTION  = process?.env?.NODE_ENV
```

which would return `false` if any of those where undefined or null.

Reply:
Another option could be simply to set the default level as `0` and make the user set it to `1` when they want to see the output.